src/plugins/kibana/public/settings/sections/indices/add_data_steps/pattern_review_step.html

```diff
@@ -1,7 +1,28 @@
-<h2>Pattern review step</h2>
+<div class="wizard-step-title">
+  <h3>Review the index pattern</h3>
+  Here we'll define how and where to store your parsed events. We've made some intellient guesses for you, but most
+  fields can be changed if we got it wrong!
+</div>
 
-<div>
-  Docs: {{sampleDocs}}
+<div class="pattern-review">
+  <input ng-model="reviewStep.indexPattern.id" class="pattern-input"/>
```

Comment:
1. thoughts on a label for this saying it's the index pattern name?  
2. should this be required before the save button is enabled?
3. if 2, thoughts on making this a form and using angular's validation?